crates/next-dev/tests/integration.rs

```diff
@@ -131,6 +132,19 @@ async fn run_test(resource: &str) -> JestRunResult {
     let project_dir = workspace_root.join(resource);
     let requested_addr = get_free_local_addr().unwrap();
 
+    // Reference this here so it's dropped in this scope along with the browser and
+    // Turbopack server
+    let mut _font_server: Option<MockServer> = None;
+    let font_dir = path.join("__google_fonts__");
+    if font_dir.exists() {
+        let server = create_font_server(&font_dir);
+        env::set_var(
+            "TURBOPACK_TEST_ONLY_GOOGLE_FONTS_STYLESHEET_URL",
```

Comment:
I _think_ this should be okay with the way `cargo test`/`cargo nexttest` run tests in parallel, with multiple threads but only one test running in serial within the thread (see unsetting this value below)